dagster_ray/executor.py

```diff
@@ -23,7 +23,12 @@
     StepHandler,
     StepHandlerContext,
 )
-from dagster._core.remote_representation.origin import RemoteJobOrigin
+
+try:
+    from dagster._core.remote_representation.origin import RemoteJobOrigin
+except ImportError:
+    # for new versions of dagster > 1.11.6
+    from dagster._core.remote_origin import RemoteJobOrigin  # type: ignore[reportMissingImports]
```

Comment:
try `pyright` instead of `type`

Reply:
addressed now - tests are mostly green can I re-trigger the 1 red one? I think it is a flaky one

Reply:
`TimeoutError: Failed to get head node agent within 10 seconds. ... It's possible that the agent didn't launch successfully due to port conflict.`